README.md

```diff
@@ -172,13 +172,56 @@ docker -v ./$TESTNET_NAME:/testnet -p 9000:9000/tcp -p 9000:9000/udp -p 8008:800
 
 Note: `ETH1_ENDPOINT` should be figured out locally as it depends on OS and Eth1 node setup.
 
+#### Prysm
+
+For now, Prysm provides instructions for building from source, but in the future can provide binaries and Docker containers for the merge testnet.
+
+Required dependencies:
+
+- UNIX operating system
+- The latest release of [Bazel](https://docs.bazel.build/versions/4.0.0/install.html) installed
+- The `cmake` package installed
+- The git package installed
+- `libssl-dev` installed
+- `libgmp-dev` installed 
+- `libtinfo5` installed
+
+Then, clone the repo and checkout the branch
+
+```shell
+git clone -b merge https://github.com/prysmaticlabs/prysm.git clients/prysm && cd clients/prysm
+```
+
+Build the beacon node and validator
+
+```shell
+bazel build //beacon-chain:beacon-chain
+bazel build //validator:validator
+```
+
+Run the beacon node
+
+```shell
+bazel run //beacon-chain --define=ssz=minimal -- \
+ --datadir="./$TESTNET_NAME/nodes/prysm0/beacondata" \
+ --min-sync-peers=0 \
+ --http-web3provider=http://127.0.0.1:8545 \
+ --bootstrap-node= \
+ --chain-config-file="./$TESTNET_NAME/public/eth2_config.yaml" \
+ --genesis-state="./$TESTNET_NAME/public/genesis.ssz" \
+ --force-clear-db
```

Comment:
`--force-clear-db` doesn't seem right, you lose all data on a restart this way. It may help when re-syncing an eth1 node for testing, but shouldn't be the default.